rir/src/api.cpp

```diff
@@ -66,7 +66,7 @@ REXPORT SEXP rir_disassemble(SEXP what, SEXP verbose) {
     return R_NilValue;
 }
 
-REXPORT SEXP rir_compile(SEXP what, SEXP env) {
+REXPORT SEXP rirCompile(SEXP what, SEXP env) {
```

Comment:
Currently all the symbols that we call from R (i.e. all functions in api.cpp) have this `rir_` prefix. it seems strange only to change one...